src/Bicep.MSBuild.E2eTests/package.json

```diff
@@ -20,6 +20,7 @@
   "scripts": {
     "compile": "tsc -p ./",
     "test": "jest",
+    "test:ado": "jest --config jest.config.ado.js",
```

Comment:
Is there going to be a corresponding change in ADO to specifically run this script?

Reply:
Yes, [this line](https://msazure.visualstudio.com/One/_git/BicepMirror?path=/.pipelines/PackagesTest.yml&version=GBmain&line=65&lineEnd=66&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) has to be changed to `npm run test:ado`.